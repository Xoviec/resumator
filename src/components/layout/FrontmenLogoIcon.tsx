import { SvgIcon, SvgIconProps } from "@mui/material";
import { VoidFunctionComponent } from "react";

export const FrontmenLogoIcon: VoidFunctionComponent<SvgIconProps> = (props) => {
  return (
    <SvgIcon viewBox="0 0 96 96" {...props}>
      <g fill="none" fillRule="evenodd" transform="translate(7, 0)">
        <path fill="#FF5900" d="M43 46.04v14.694l26-14.693V96l13-7.347V24z" />
        <path
          fill="#0CC"
          d="M40.5 0L5 20.633 17.679 28 40.5 14.737 63.322 28 76 20.633z"
        />
        <path
          fill="#FFF"
          d="M0 88.653L13 96V72.493L31.2 82.78V68.085L13 57.8V46.004l26 14.731V46.041L0 24z"
        />
      </g>
    </SvgIcon>
  );
};
