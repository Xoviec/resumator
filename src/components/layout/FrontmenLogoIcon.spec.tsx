import { render } from "@testing-library/react";

import { FrontmenLogoIcon } from "./FrontmenLogoIcon";

describe("FrontmenLogoIcon", () => {
  it("renders correctly", () => {
    const { container } = render(<FrontmenLogoIcon />);
    expect(container).toMatchSnapshot();
  });
});
