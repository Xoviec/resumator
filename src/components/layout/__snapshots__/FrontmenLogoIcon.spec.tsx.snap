// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrontmenLogoIcon renders correctly 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
    focusable="false"
    viewBox="0 0 96 96"
  >
    <g
      fill="none"
      fill-rule="evenodd"
      transform="translate(7, 0)"
    >
      <path
        d="M43 46.04v14.694l26-14.693V96l13-7.347V24z"
        fill="#FF5900"
      />
      <path
        d="M40.5 0L5 20.633 17.679 28 40.5 14.737 63.322 28 76 20.633z"
        fill="#0CC"
      />
      <path
        d="M0 88.653L13 96V72.493L31.2 82.78V68.085L13 57.8V46.004l26 14.731V46.041L0 24z"
        fill="#FFF"
      />
    </g>
  </svg>
</div>
`;
